app/javascript/mastodon/components/autosuggest_textarea.js

```diff
@@ -18,11 +20,11 @@ const textAtCursorMatchesToken = (str, caretPosition) => {
     word = str.slice(left, right + caretPosition);
   }
 
-  if (!word || word.trim().length < 2 || word[0] !== '@') {
+  if (!word || word.trim().length < 2 || ['@', ':'].indexOf(word[0]) === -1) {
```

Comment:
`!['@', ':'].includes(word[0])`

Reply:
I'm sorry but I'm really used to the old way... It's not really important is it? We don't have a code style for includes vs indexOf

Reply:
includes is short and simple. However, it is not important.